vividus-plugin-html/src/main/java/org/vividus/bdd/steps/html/HtmlSteps.java

```diff
@@ -56,6 +48,36 @@
         return Optional.empty();
     }
 
+    /**
+     * Checks if HTML contains elements according to rule by CSS selector
+     * @param cssSelector CSS selector
+     * @param html HTML to check
+     * @param comparisonRule The rule to compare values
+     * <br>(<i>Possible values:</i>
+     * <br> less than (&lt;),
+     * <br> less than or equal to (&lt;=),
+     * <br> greater than(&gt;)
+     * <br> greater than or equal to(&gt;=),
+     * <br> equal to(=))
+     * @param number Number of elements
+     * @see <a href="https://www.w3schools.com/cssref/css_selectors.asp"><i>CSS Selector Reference</i></a>
+     * @see <a href="https://jsoup.org/apidocs/org/jsoup/select/Selector.html"><i>Jsoup Selector API</i></a>
+     * @return true if elements quantity corresponds to rule, false otherwise
+     */
+    @Then("number of elements found by CSS selector `$cssSelector` in HTML `$html` is $comparisonRule `$quantity`")
+    public boolean doesElementByCssSelectorExist(String cssSelector, String html, ComparisonRule comparisonRule,
+            int number)
+    {
+        Elements elements = findElements(html, cssSelector);
+        return softAssert.assertThat(String.format("Number of elements found by CSS selector '%s' is %s %d",
+                cssSelector, comparisonRule, number), elements.size(), comparisonRule.getComparisonRule(number));
```

Comment:
do you really need to add `comparisonRule` and `number` to assertion description? it could be excessive since similar piece of information is added by Matcher

Reply:
I think we should pick one solution. BaseValidation uses this approach.
If we will rely on matcher, the PR is obsolete, but we should refactor basevalidations' code